wiki/Tournaments/MWC/2021_4K/en.md

```diff
@@ -59,71 +59,81 @@ The osu!mania 4K World Cup 2021 is run by the [osu! team](/wiki/People/The_Team)
 1. The osu!mania 4K World Cup is a country-based team tournament, played on the osu!mania game mode.
    - *Notice: While this competition has been planned as a 3 versus 3 setup, this might change depending on the amount of registrations.*
 2. Beatmap scoring will be based on Score V2.
-3. The beatmaps for each round will be announced by the Map Selector in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
-   - One beatmap will be given as a tiebreaker beatmap. This beatmap will only be played in case of a tie.
-4. Match schedule will be settled by the Tournament Management (see below).
-5. If no staff or referee is available, the match will be postponed.
+3. The beatmaps for each round will be announced by the Mappool Selector Team on the Sunday before the actual matches take place.
+   - Each mappool, with the exception of the Qualifier stage, will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. 4-4 in a best of nine.
```

Comment:
```suggestion
   - Each mappool, with the exception of the Qualifier stage, will contain a *tiebreaker* beatmap. It will only be played in case of a tie in the scoreline in the *best of* system, e.g. 4-4 in a best of 9.
```